e2e/specs/st_set_page_config.spec.js

```diff
@@ -45,4 +45,10 @@ describe("st.set_page_config", () => {
       "wide"
     );
   });
+
+  it("displays in wide mode", () => {
+    cy.get("[data-testid='stReportViewContainer']").matchImageSnapshot(
+      "wide-mode"
+    );
+  });
```

Comment:
You need to add an image snapshot. You can pull it from CircleCI job

Reply:
Ah, thanks for the clarification! I kinda assumed it would get created automatically once I pushed, didn't realize I have to manually copy it over as well.